dev-libs/rocksdb/rocksdb-9.6.1.recipe

```diff
@@ -91,11 +102,11 @@ INSTALL()
 		$libDir/cmake
 
 	# tools package
-	mkdir -p $binDir
-	cp cache_bench $binDir
-	cp db_bench $binDir
-	cp tools/ldb $binDir
-	cp tools/sst_dump $binDir
+	mkdir -p $commandBinDir
+	cp cache_bench $commandBinDir/cache_bench$commandSuffix
+	cp db_bench $commandBinDir/db_bench$commandSuffix
+	cp tools/ldb $commandBinDir/ldb$commandSuffix
+	cp tools/sst_dump $commandBinDir/sst_dump$commandSuffix
```

Comment:
I think `$commandSuffix` isn't needed here. It would be added by haikuporter automatically if there were one (which is the case by default, see the default value in line 26 above). It is only needed for the PROVIDES.

Reply:
Fixed.